tools/scripts/generate.sh

```diff
@@ -34,6 +34,7 @@ npm exec openapi-generator-cli -- generate \
     --package-name="$client_package" \
     --type-mappings=integer=int \
     --type-mappings=object=interface{} \
+    --type-mappings=interface{}=any \
```

Comment:
important to do it in a line, if the change is:
```
--type-mappings=object=any \
```

then it'll create *any (pointer to any) for response params